src/library/scala/collection/Iterator.scala

```diff
@@ -826,6 +826,15 @@ trait Iterator[+A] extends IterableOnce[A] with IterableOnceOps[A, Iterator, Ite
     }
   }
 
+  /** Checks whether corresponding elements of the given iterable collection
+   *  compare equal (with respect to `equals`) to elements of this $coll.
```

Comment:
```suggestion
   *  compare equal (with respect to `==`) to elements of this $coll.
```

Reply:
Or do we take "equals" to mean "==" and say something like "Object#equals" for the original one?

Reply:
That's a good question. Selecting equals was the basis for https://github.com/scala/scala/pull/9349

We should have words to say `Any_equals`.

Reply:
I like `==` and "equals" because it's the user-facing verbiage.  Or "it's what I grew up calling them".